internal/pkg/keeper/consts.go

```diff
@@ -0,0 +1,18 @@
+//
+// Copyright (C) 2024 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package keeper
+
+const (
+	ApiVersion                      = "v3"
+	ApiBase                         = "/api/v3"
+	ContentType                     = "Content-Type"
+	ContentTypeJSON                 = "application/json"
+	ContentTypeText                 = "text/plain"
+	ApiPingRoute                    = ApiBase + "/ping"
+	ApiRegisterRoute                = ApiBase + "/registry"
+	ApiAllRegistrationRoute         = ApiRegisterRoute + "/all"
+	ApiRegistrationByServiceIdRoute = ApiRegisterRoute + "/serviceId/"
```

Comment:
@jackchenjc, @cloudxxx8 , @jumpingliu , There are trade-offs to avoiding being dependent on Core Contracts for many of these constants, Base DTOs and sharing the new request/response DTOs with Core Keeper in edgex-go. Now that we have code that highlights the amount of duplication required to avoid dependency, I think the WG should revisit this and decide if avoiding the duplicate code outweighs the additional dependency.

Reply:
@lenny-intel I agree with you, and I am asking Jack to move those code from contracts to here